test/Microsoft.AspNetCore.Razor.Evolution.Test/Microsoft.AspNetCore.Razor.Evolution.Test.csproj

```diff
@@ -3,23 +3,18 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
     <DefineConstants>$(DefineConstants);__RemoveThisBitTo__GENERATE_BASELINES</DefineConstants>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs" Exclude="TestFiles\**\*" />
-    <EmbeddedResource Include="**\*.resx" />
+    <Compile Include="**\*.cs" Exclude="TestFiles\**\*;$(DefaultItemExcludes)" />
```

Comment:
We should be able to avoid this by appending `TestFiles\**\*` to ` `$(DefaultItemExcludes)`